src/neuroglancer/widget/shader_code_widget.ts

```diff
@@ -16,7 +16,7 @@
 
 import 'codemirror/addon/lint/lint.js';
 
-import CodeMirror from 'codemirror';
```

Comment:
Please also revert this unrelated change.  Depending on how the typings are defined, how webpack is configured, and in what mode typescript is run, one or the other import style is needed.  However, I believe the existing style is what works best with the current neuroglancer configuration.